### PR TITLE
Sheets for non-stowing container show should not show form elements for reducing bulk

### PR DESCRIPTION
**Pull request by maplealmond**
Addresses: https://github.com/foundryvtt/pf2e/issues/14073

The page for containers shows reduced bulk even if the container is non-stowing.  The definition of a non-stowing container within the system is a "not real" container such as a belt, where the items are functionally "worn" as opposed to "stowed".

The FVTT PF2e system does not allow non-stowing containers to reduce bulk, so the form element is confusing.  This causes them to be hidden, so the user has greater clarity that these numbers will not apply.  The stowing container checkbox is moved upwards as the relevance of the maximum and ignored bulk elements is subordinate to the existence of the stowing field.

https://github.com/foundryvtt/pf2e/assets/22515/6083e8f7-a7c3-40a4-b388-33c183d160da

### CODE REVIEW

**Comment by stwlam** [comment]
Have it clean up after itself if not stowing (hidden fields may be sufficient)

**Comment by maplealmond** [comment]
Apologies, I think I might need a bit more detail to understand the request.  It's not clear to me what needs to be cleaned up. 

Should the Bulk Capacity and Bulk Ignored fields be cleared if the contained is marked non-stowing?

**Comment by CarlosFdez** [comment]
I think the normal approach we take is to override _preUpdate() in order to do it.

**Comment by maplealmond** [comment]
@CarlosFdez That's a good pointer, but I'm still taking a bit of a guess at what the request is.  

The values in question being hidden are non nullable numeric, so I have adjusted this PR so that on a non-stowing container they are zeroed.  If this is not what you mean @stwlam let me know and I'll adjust accordingly.

File: static/templates/items/backpack-details.hbs [hunk 1]
```diff
@@ -17,7 +24,4 @@
     <input type="number" name="system.bulk.ignored" id="{{fieldIdPrefix}}ignored" value="{{data.bulk.ignored}}" />
 </div>
```
Comment (CarlosFdez) [inline static/templates/items/backpack-details.hbs:25]: You'll want to indent these form groups and remove the empty space after the if and before the end/if to maintain convention.

**Comment by CarlosFdez** [comment]
Other than what I mentioned it looks fine. I think clearing out the values might run into some quirks if someone toggles the property on an existing bag trying to see what it does and then has to put the value back in again, but that is a minor thing.

File: static/templates/items/backpack-details.hbs (hunk 1 above)
Comment (maplealmond) [reply static/templates/items/backpack-details.hbs]: Done.

**Comment by TikaelSol** [comment]
You can run `npm run lint --fix` to fix most formatting issues that prettier flags automatically. After that `npm run test` will run the tests locally to make sure the pipeline will pass here on github

**Pull request merged**